mvnw

```diff
@@ -0,0 +1,234 @@
+#!/bin/sh
```

Comment:
if it's easy, can you add a pointer somewhere (in the README?) about how this was generated?